pyproject.toml

```diff
@@ -30,6 +30,9 @@ google-api-python-client = "^2.9.0"
 google-oauth = "^1.0.0"
 google-auth-httplib2 = "^0.1.0"
 google-auth-oauthlib = "^0.4.4"
+# Added pinned typed-add to overcome issues with black version
+typed-ast = "^1.5.0"
```

Comment:
If this is a Black requirement, it should be a dev dependency not a main dependency.